plugins/systemstat.koplugin/main.lua

```diff
@@ -0,0 +1,80 @@
+
+local KeyValuePage = require("ui/widget/keyvaluepage")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local _ = require("gettext")
+
+local SystemStat = {
+    start_sec = os.time(),
+    wakeup_count = 0,
+    sleep_count = 0,
+    charging_count = 0,
+    decharging_count = 0,
+}
+
+function SystemStat:onSuspend()
+    self.sleep_count = self.sleep_count + 1
+end
+
+function SystemStat:onResume()
+    self.wakeup_count = self.wakeup_count + 1
+end
+
+function SystemStat:onCharging()
+    self.charging_count = self.charging_count + 1
+end
+
+function SystemStat:onNotCharging()
+    self.decharging_count = self.decharging_count + 1
+end
+
+function SystemStat:onCallback()
+    local kv_pairs = {
+        {_("Startup at"), os.date("%c", self.start_sec)},
+        {_("Running hours"),
```

Comment:
How geeky is uptime? How about "Operating time"? I'm just not sure how clear "running hours" is. What does Win/Mac say?